content/en/tutorials/vpc_peering.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "AWS VPC Peering"
+linkTitle: "AWS VPC Peering"
+date: 2021-07-21
+draft: false
+description: Guide on how to peer AWS VPCs
+---
+
+## What is VPC Peering?
+VPC peering is the standard way of opening _private_ network communication between 2 AWS VPCs. This means a server in
+one VPC can send requests to a private IP (e.g. another server, a database, etc...) in the other network, without
+touching the public internet. This ability is extremely useful if one wishes to connect different Opta environments
+(or a pre-existing VPC and an Opta environment) together without any serious security risk. This guide will show
+you step-by-step instructions on how to do it.
+
+## Limitation?
+VPCs can peer from cross AWS account and even cross region with minimal difficulties, but they can never peer
+with another VPC that shares an overlapping CIDR block.
+
+Furthermore, AWS security groups do not work across VPC peerings-- you will need to whitelist by individual IPs or CIDR
+blocks.
+
+## Send the VPC Peering
+The first step in peering a VPC is to send the peering request on behalf of one of them. It does not matter which VPC
+initiates as the peering described here will be symmetric. To do so, go to the AWS UI console on your web browser,
+open the VPC service homepage, and on the left bar search for "Peering Connections". CLick it
+
+![image alt text](/images/aws_peering_1.png)
```

Comment:
you can make this image clickable like this https://github.com/run-x/opta-docs/blob/main/content/en/Architecture/aws.md?plain=1#L13-L15

and do the same for all the image in this page, since they could be hard to read right now